segmatch/include/segmatch/opencv_random_forest.hpp

```diff
@@ -2,17 +2,14 @@
 #define SEGMATCH_OPENCV_RANDOM_FOREST_HPP_
 
 #include <nabo/nabo.h>
-#include <opencv2/core/core.hpp>
-#include <opencv2/ml/ml.hpp>
 
 #include "segmatch/common.hpp"
 #include "segmatch/parameters.hpp"
 #include "segmatch/segmented_cloud.hpp"
 
 namespace segmatch {
 
-//TODO(Renaud or Daniel) Implement a base class when another classifier should be
-// evaluated
+//TODO Renaud matching needs major refactoring.
 class OpenCvRandomForest {
```

Comment:
I see that the whole class need mayor refactoring anyway, but I for publication I think that we should at least rename it to something more meaningful (ah, I wanted to it since so long...). What about something like NearestNeighborsDescriptorsMatcher (which could nicely inherit from an abstract class DescriptorsMatcher)?

Reply:
Yes I totally agree! Wanna give it a shot? Otherwise I can do it before we merge.

Reply:
At this point I believe that there is no need for a base class though.

Reply:
I'm sorry, I don't have the repo anymore, had to clean up my machines :P Should be just a matter of renaming the class, the file and change the cake. Yes of course now it's not necessary, you can introduce it if/when you add new matching strategies!

Reply:
Ok I'll add it to the todos and make sure to change the *cake* ;)